src/models/Tasks.js

```diff
@@ -1,8 +1,11 @@
 import mongoose from "mongoose"
-// const mongoose = require("mongoose")
 
 const TaskSchema = mongoose.Schema(
     {
+        _id: {
+            type: String,
+            required: false,
+        },
```

Comment:
updated TaskSchema to allow for an optional id to be passed; If it's not included, MongoDB will auto assign an _id

Should test for this....